image_data_guide/04b_tensorflow_training.ipynb

```diff
@@ -0,0 +1,889 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow Training with SageMaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download | Structure | Preprocess | **Train Model**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Somewhere explain how this notebook differs from 04b and 04c. e.g."In this notebook you will go through and train a model using the image data you processed earlier in this notebook series. This notebook will show you how to train a pre-built algorithm provide by AWS SageMaker..."

Reply:
added notes on top about the notebook's context in the series